python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -1152,6 +1188,10 @@ def submit_run_request(
 
         check_for_debug_crash(debug_crash_flags, "RUN_IDS_ADDED_TO_EVALUATIONS")
 
-        tick_context.update_state(
-            TickStatus.SUCCESS if len(run_requests) > 0 else TickStatus.SKIPPED,
-        )
+        if user_canceled:
+            # mark as skipped so that we don't request any remaining runs when the sensor is started again
+            tick_context.update_state(TickStatus.SKIPPED)
```

Comment:
@OwenKephart we talked about potentially setting a boolean on the tick data that the tick was manually stopped. Then we'd check that value when we decide if a tick should be retried. We already have logic in place to not retry SKIPPED steps, and we'd end up marking this tick as SKIPPED once we decide not to retry it, so i figured it was simpler to just mark it skipped right away. It has the additional benefit of not having a tick hanging in STARTED status while the sensor is off

Reply:
Ah interesting -- I think using "SKIPPED" to mark this state makes sense but agree that even if it's just for internal use right now it'd be good to have some way of differentiating this case

Reply:
>  it incorrectly say that certain things were launched that were not?

I'm not sure, but will definitely test

Reply:
> even if it's just for internal use right now it'd be good to have some way of differentiating this case

sounds good, will add

Reply:
did you verify that the UI actually shows things getting launched here? Just wanted to make sure that it doesn't assume that SKIPPED is synonymous with no runs

Reply:
Ran a test case and i am seeing some UI things we will need to fix - 
1. There are a couple places where we assume SKIPPED = 0 requests in the UI. Those were easy enough to (kind of) fix, but then 
2. The UI is oriented around "requested materializations" not the materializations that actually had runs submitted. For successful ticks, those are equivalent, but for these terminated ticks, the list of requested materializations is a superset of the materializations that were actually run. I have a couple ideas for how to handle this, i'll put together a rough UI pr to get some feedback